scanpipe/migrations/0068_merge_20240820_1656.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 5.0.7 on 2024-08-20 16:56
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('scanpipe', '0067_discoveredpackage_notes'),
+        ('scanpipe', '0067_packagescore_scorecardcheck'),
+    ]
+
+    operations = [
+    ]
```

Comment:
The migrations file need to be consolidated.

Migrate your local db to 0066: `./manage.py migrate scanpipe 0066`

Delete those migrations files (0067 and 0068) and regenerate a single clean migration file.